>From 3f38064fb7548a9b3c489befc5d3eeeb9b29b8e8 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 30 Nov 2011 11:02:20 +0200
Subject: [PATCH 1/3] man/dpkg-architecture.1: (COMMANDS, OPTIONS): Order
 items alphabetically
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 man/dpkg-architecture.1 |   28 ++++++++++++++--------------
 1 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/man/dpkg-architecture.1 b/man/dpkg-architecture.1
index bf27079..fc7c32e 100644
--- a/man/dpkg-architecture.1
+++ b/man/dpkg-architecture.1
@@ -2,7 +2,7 @@
 .SH "NAME"
 dpkg\-architecture \- set and determine the architecture for package building
 .
-.SH SYNOPSIS
+ajor.SH SYNOPSIS
 .B dpkg\-architecture
 .RI [ option "...] [" command ]
 .PP
@@ -25,9 +25,9 @@ does not match the default.
 .
 .SH COMMANDS
 .TP
-.B \-l
-Print the environment variables, one each line, in the format
-\fIVARIABLE=value\fP. This is the default action.
+.BI \-c " command"
+Execute a \fIcommand\fP in an environment which has all variables set to
+the determined value.
 .TP
 .BI \-e debian-architecture
 Check for equality of architecture. By default \fIdebian-architecture\fP
@@ -41,6 +41,13 @@ as an architecture wildcard and comparing against the current Debian
 architecture. Command finishes with an exit status of 0 if matched, 1 if
 not matched.
 .TP
+.B \-l
+Print the environment variables, one each line, in the format
+\fIVARIABLE=value\fP. This is the default action.
+.TP
+.B \-L
+Print a list of valid architecture names.
+.TP
 .BI \-q variable-name
 Print the value of a single variable.
 .TP
@@ -51,13 +58,6 @@ using eval.
 .B \-u
 Print a similar command to \fB\-s\fP but to unset all variables.
 .TP
-.BI \-c " command"
-Execute a \fIcommand\fP in an environment which has all variables set to
-the determined value.
-.TP
-.B \-L
-Print a list of valid architecture names.
-.TP
 .B \-\-help
 Show the usage message and exit.
 .TP
@@ -69,15 +69,15 @@ Show the version and exit.
 .BI \-a debian-architecture
 Set the Debian architecture.
 .TP
-.BI \-t gnu-system-type
-Set the GNU system type.
-.TP
 .B \-f
 Values set by existing environment variables with the same name as used by
 the scripts are honored (i.e. used by \fBdpkg\-architecture\fP), except if
 this force flag is present. This allows the user
 to override a value even when the call to \fBdpkg\-architecture\fP is buried
 in some other script (for example \fBdpkg\-buildpackage\fP(1)).
+.TP
+.BI \-t gnu-system-type
+Set the GNU system type.
 .
 .SH TERMS
 .IP "build machine" 4
-- 
1.7.7.1

